presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestRemoveEmptyDelete.java

```diff
@@ -44,7 +45,10 @@ public void testDoesNotFire()
                                         TestingTransactionHandle.create(),
                                         Optional.empty()),
                                 ImmutableList.of(),
-                                ImmutableMap.of()),
+                                ImmutableMap.of(),
+                                false,
+                                TupleDomain.all(),
```

Comment:
Why adding these two `TupleDomain.all()` here? The other constructor has the `delete` field as well

Reply:
no it doesn't. This is the planbuilder constructor, and I only added isDelete to the constructor that too all arguments.  I didn't want to make everything constructing a tablescan in a test need to pass in whether it's a delete, since it's very commonly used, and this was the only test that actually needed it.